### PR TITLE
Fix tooltip issues again maybe

### PR DESCRIPTION
**Pull request by ozerovs**

### CODE REVIEW

File: packages/components/src/base_tooltip/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  // Fix to avoid jumping during tooltip recalculation
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  z-index: -1;
+  visibility: hidden;
+
+  &.isActive {
+    z-index: 100;
```
Comment (peterhass) [inline packages/components/src/base_tooltip/index.module.scss:11]: I'm not sure we need a z-index that high. To be honest, I am still not quite too sure about how z-index works. We use z-index: 9 in modals. I wouldn't want feature alerts overlapping modals. Can you check if something like `3` or `4` would work as well?

File: packages/components/src/feature_alert/index.jsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ const FeatureAlertTooltip = (props) => {
       <BaseTooltip
         bubble={bubble} position={position}
         tooltipClassName={tooltipClassName}
-        arrowClassName={styles.arrow}
+        arrowClassName={styles.arrow} active={isOpen}
```
Comment (peterhass) [inline packages/components/src/feature_alert/index.jsx:79]: straighten naming: either use `active` or `open`

File: packages/components/src/feature_alert/index.module.scss [hunk 3]
```diff
@@ -16,17 +16,10 @@ $feature-alert-arrow-size: 16px;
   width: 300px;
   position: absolute;
   cursor: pointer;
-  z-index: -1;
 
   animation-duration: 300ms;
   animation-name: feature-alert-appear;
   animation-timing-function: ease-in;
```
Comment (peterhass) [inline packages/components/src/feature_alert/index.module.scss:22]: does this thing still work?

File: packages/components/src/tooltip/index.jsx [hunk 4]
```diff
@@ -13,9 +13,14 @@ const Tooltip = (props) => {
     ...cleanProps
   } = props;
 
+  const [isActive, setActive] = useState(false);
+
   return (
     <BaseTooltip
       {...cleanProps}
+      onMouseEnter={() => setActive(true)}
+      onMouseLeave={() => setActive(false)}
```
Comment (peterhass) [inline packages/components/src/tooltip/index.jsx:22]:
```suggestion
      onMouseEnter={setActive.bind(undefined, true)}
      onMouseLeave={setActive.bind(undefined, false)}
```

?

File: packages/components/src/feature_alert/index.module.scss (hunk 3 above)
Comment (ozerovs) [reply packages/components/src/feature_alert/index.module.scss:22]: It does, why?

File: packages/components/src/base_tooltip/index.module.scss (hunk 1 above)
Comment (ozerovs) [reply packages/components/src/base_tooltip/index.module.scss:11]: Will put 8 for the sake of modals.

**Review by peterhass — APPROVED** [review]

**Pull request merged**